lib/db/mysql.js

```diff
@@ -363,32 +392,60 @@ module.exports = function (log, error) {
     })
   }
 
-  // Select : devices d, sessionTokens s, accounts a
+  // Since MySQL doesn't have an ARRAY type, we store arrays as comma-separated
```

Comment:
More bike-shedding, but this comment reads a little bit inaccurately to me. Unless I misunderstand, we're not *storing* them as comma-separated values, they're stored as rows in `devicesCapabilities`. They're just returned from queries as comma-separated values, right?